### ISSUE TITLE
Fish support?

### ISSUE THREAD

**Issue by millsjoe** [petobens/trueline#3]
Not an issue, more a request (as per Reddit).

Great on Bash would love it on Fish 😄

**Comment by petobens** [comment]
Does something similar for fish exist already (I'm not a fish user myself)? For instance I haven't added zsh support (yet) because there are plenty of powerline-like prompts out there (powerlevel9k, powerlevel10k, etc)

**Comment by millsjoe** [comment]
Yeah there exists `Oh my fish` which is similar. (I recommend giving fish a try and see if it appeals to you). I get what you mean. I think there is actually an `Oh my zsh` extension too. Nevertheless, great job on this 👍

**Comment by petobens** [comment]
Ok, let's leave this open for a while though.
